pkg/apiserver/apiserver.go

```diff
@@ -174,7 +174,8 @@ func InstallVersionHandler(mux Mux, container *restful.Container) {
 			Doc("get the code version").
 			Operation("getCodeVersion").
 			Produces(restful.MIME_JSON).
-			Consumes(restful.MIME_JSON))
+			Consumes(restful.MIME_JSON).
+			Writes(version.Info{}))
```

Comment:
Do we really need this change? with the latest code in  curl -kvs http://127.0.0.1:8080/version already prints the version numbers

```
$ curl -kvs http://127.0.0.1:8080/version
*   Trying 127.0.0.1...
* Connected to 127.0.0.1 (127.0.0.1) port 8080 (#0)
> GET /version HTTP/1.1
> Host: 127.0.0.1:8080
> User-Agent: curl/7.47.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Content-Type: application/json
< Date: Mon, 06 Jun 2016 13:21:42 GMT
< Content-Length: 295
<
{
  "major": "1",
  "minor": "3+",
  "gitVersion": "v1.3.0-alpha.4.1106+68097515d82121-dirty",
  "gitCommit": "68097515d821217918a1c916759a8373e5e5eb86",
  "gitTreeState": "dirty",
  "buildDate": "2016-06-06T13:18:29Z",
  "goVersion": "go1.6.1",
  "compiler": "gc",
  "platform": "linux/amd64"
* Connection #0 to host 127.0.0.1 left intact
}
```